community/kernel/src/test/java/org/neo4j/test/subprocess/SubProcess.java

```diff
@@ -214,7 +197,7 @@ private String classPath( String parentClasspath )
 
     private static Process start( String... args )
     {
-        ProcessBuilder builder = new ProcessBuilder( args );
+        ProcessBuilder builder = new ProcessBuilder( args ).inheritIO();
```

Comment:
We could alternatively merge the PR and see what breaks. Again. It might be that those tests have already been changed or removed, or we might find that those tests should be changed or removed. How does that sound, @thobe ?